src/drm/steam_drm.cpp

```diff
@@ -30,18 +29,18 @@ void SteamDRM::checkExists()
 #elif defined(__APPLE__)
     steamFolder = QDir(QDir::home().filePath("Library/Application Support/Steam"));
 #endif
+    QDir steamAppsDir = steamFolder.filePath("steamapps");
+    if (!steamAppsDir.exists())
+    {
+        steamAppsDir = steamFolder.filePath("SteamApps");
+    }
 
-    if (steamFolder.filePath("").trimmed() != "" && steamFolder.exists() && steamFolder != QDir("."))
+    if (QDir(steamAppsDir.filePath("libraryFolders.vdf")).exists() && steamFolder.filePath("").trimmed() != "" && steamFolder.exists() && steamFolder != QDir("."))
```

Comment:
You're right, don't commit at 3am xD

On 3 February 2016 05:44:18 GMT+00:00, Jake Schadewe <notifications@github.com> wrote:
>>  
>> -    if (steamFolder.filePath("").trimmed() != "" &&
>steamFolder.exists() && steamFolder != QDir("."))
>> +    if (QDir(steamAppsDir.filePath("libraryFolders.vdf")).exists()
>&& steamFolder.filePath("").trimmed() != "" && steamFolder.exists() &&
>steamFolder != QDir("."))
>
>Should be QFile.exists() in the first conditional here.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/Proj-Ascension/Client/pull/118/files#r51680955

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.